src/blocks/IconForestBlock.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import {
+  ContentWrapper,
+  SectionWrapper,
+  TABLET_BP_DOWN,
+} from '../components/blockHelpers'
+import { BaseBlockProps, MarkdownHtmlComponent } from './BaseBlockProps'
+
+const ColumnContentWrapper = styled(ContentWrapper)({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  textAlign: 'center',
+})
+
+const Title = styled('h2')({
+  fontSize: '3.75rem',
+  width: '100%',
+  margin: 0,
+  [TABLET_BP_DOWN]: {
+    fontSize: '2.25rem',
+  },
+})
+
+const Picture = styled('picture')({
+  maxWidth: '47.125rem',
+  marginTop: '3.125rem',
+  [TABLET_BP_DOWN]: {
+    maxWidth: '100%',
+    width: 'auto',
+    display: 'block',
+  },
+})
+
+const Paragraph = styled('div')({
+  display: 'block',
+  fontSize: '1.125rem',
```

Comment:
are we sure we want `1.125rem`s here? Considering the rest of the site uses `16px` instead of `18px` font size i think it'll look weird if we make this particular section `18px`?